core/dictgen/src/rootcling_impl.cxx

```diff
@@ -3707,8 +3707,7 @@ gOptCint("cint", llvm::cl::desc("Deprecated, legacy flag which is ignored."),
         llvm::cl::Hidden,
         llvm::cl::cat(gRootclingOptions));
 static llvm::cl::opt<bool>
-gOptReflex("reflex", llvm::cl::desc("Deprecated, legacy flag which is ignored."),
-          llvm::cl::Hidden,
+gOptReflex("reflex", llvm::cl::desc("Behave internally like genreflex."),
```

Comment:
The previous text was saying that the option was ignored.  Does this PR actually implement the "Behave internally like genreflex." or was the hold text inaccurate?

Reply:
It implemented that behavior long time ago and did not update the documentation…